src/test/java/core/basesyntax/service/RegistrationServiceTest.java

```diff
@@ -0,0 +1,119 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        user = new User();
+    }
+
+    @BeforeEach
+    void setUp() {
+        user = new User();
+        user.setLogin("usersLogin");
+        user.setPassword("usersPassword");
+        user.setAge(18);
+    }
+
+    @Test
+    void register_login_Ok() {
```

Comment:
each part should be in lower case
```suggestion
    void register_login_ok() {
```